### PR TITLE
LPS-158485 [Raylife AP] Create Channel Agent Portal

### PR DESCRIPTION
**Pull request by juliocezarjc**

### CODE REVIEW

**Comment by juliocezarjc** [comment]
ci:test:sf

**Comment by eliasantosf** [comment]
ci:test:sf

**Comment by joanagrego** [comment]
ci:test:sf

**Review by joanagrego — APPROVED** [review]
Nice job! :up:

**Review by eliasantosf — APPROVED** [review]
LGTM :fire: Thanks @juliocezarjc

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/contents/ProductCardPersona.tsx [hunk 1]
```diff
@@ -51,10 +51,10 @@ const ProductCardPersona = () => {
 										)}
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/contents/ProductCardPersona.tsx:51]: Please change the name of this file to ```PersonalProductCard.text``` and change this information where it is being used

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/pages/Applications.tsx [hunk 2]
```diff
@@ -50,6 +51,15 @@ const Applications = () => {
...
 
+	const callProducts = (selectCard: string) => {
+		if (cardSelected === insuranceCards[0] || selectCard === 'Personal') {
+			return <ProductCardPersona />;
+		}
+		else {
+			return <ProductCardBusiness />;
+		}
+	};
+
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/pages/Applications.tsx:62]: Please, make the change below.
```suggestion
	const callProducts = (selectCard: string) => {
		if (cardSelected === insuranceCards[0] || selectCard === 'Personal') {
			return <ProductCardPersona />;
		}
		
		return <ProductCardBusiness />;	
	};

```

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/dashboard/pages/RecentApplications.tsx [hunk 3]
```diff
@@ -201,6 +202,15 @@ const RecentApplications = () => {
...
 
+	const callProducts = (selectCard: string) => {
+		if (cardSelected === insuranceCards[0] || selectCard === 'Personal') {
+			return <ProductCardPersona />;
+		}
+		else {
+			return <ProductCardBusiness />;
+		}
+	};
+
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/dashboard/pages/RecentApplications.tsx:213]: Please, make the change below.
```suggestion
	const callProducts = (selectCard: string) => {
		if (cardSelected === insuranceCards[0] || selectCard === 'Personal') {
			return <ProductCardPersona />;
		}
		
		return <ProductCardBusiness />;
	};

```

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/commerce-catalogs/catalog.products.json [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+		"name": "productEmpty",
+		"price": "",
+		"promoPrice": "",
+		"shippable": false,
+		"sku": "RAYAP-004",
+		"specificationOptions": [
+		],
+		"taxCategory": "Normal Product",
+		"warehouse1": ""
+	},
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/commerce-catalogs/catalog.products.json:100]: Please, remove this product. It does not make sense to be added here. As a suggestion, you can talk with @gleicelisbino about this.

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/commerce-catalogs/catalog.options.json [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+			},
+			{
+				"externalReferenceCode": "INSTALLMENT",
+				"key": "installment",
+				"name": "Installment",
+				"priority": 0
+			}
+		]
+	}
+]
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/commerce-catalogs/catalog.options.json:25]: Please, check if this is necessary to be created at this moment.

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/commerce-catalogs/catalog.products.subscriptions.properties.json [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+				"subscriptionType": "monthly",
+				"subscriptionTypeSettings": {
+					"monthDay": "1",
+					"monthlyMode": "0"
+				}
+			}
+		],
+		"optionKey": "raylife-ap-option"
+	}
+]
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/commerce-catalogs/catalog.products.subscriptions.properties.json:31]: Please, check if this is necessary to be created at this moment.

**Review by silasbsi — CHANGES_REQUESTED** [review]
LGTM :1st_place_medal: But I added some suggestions.

**Comment by juliocezarjc** [comment]
ci:test:sf

**Comment by juliocezarjc** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/contents/ProductCardPersona.tsx (hunk 1 above)
Comment (juliocezarjc) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/contents/ProductCardPersona.tsx:51]: Done !

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/pages/Applications.tsx (hunk 2 above)
Comment (juliocezarjc) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/pages/Applications.tsx]: Done!

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/commerce-catalogs/catalog.products.json (hunk 4 above)
Comment (juliocezarjc) [reply modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/commerce-catalogs/catalog.products.json]: Done !

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/commerce-catalogs/catalog.products.subscriptions.properties.json (hunk 6 above)
Comment (juliocezarjc) [reply modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/commerce-catalogs/catalog.products.subscriptions.properties.json]: Done!

**Pull request closed without merging**